pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -506,6 +506,19 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                 "and no message will be printed.",
             },
         ),
+        (
+            "suggest-join-with-non-empty-separator",
+            {
+                "default": False,
```

Comment:
```suggestion
                "default": True,
```
I guess the default should stay the same than before or it's a breaking change in a minor. Waiting for another maintainer opinion. (Documentation in fragment would also be different if we change that but let's not change everything yet).